Flaky test pkg/config/tlscfg/TestReload (cert_watcher_test.go)

https://travis-ci.com/github/jaegertracing/jaeger/jobs/445878517#L2150

```
=== RUN   TestReload
    cert_watcher_test.go:110: 
        	Error Trace:	cert_watcher_test.go:110
        	Error:      	Should be true
        	Test:       	TestReload
        	Messages:   	Failed to find wanted logs. All logs: [{{info 2020-11-23 14:13:36.375563204 +0000 UTC m=+0.010957869  Loading modified certificate undefined } [{certificate 15 0 /tmp/cert.crt414800473 <nil>} {event 15 0 WRITE <nil>}]} {{error 2020-11-23 14:13:36.377159282 +0000 UTC m=+0.012553950  Failed to load certificate undefined } [{certificate 15 0 /tmp/cert.crt414800473 <nil>} {event 15 0 WRITE <nil>} {error 26 0  tls: private key does not match public key}]} {{info 2020-11-23 14:13:36.381814355 +0000 UTC m=+0.017208997  Loading modified certificate undefined } [{certificate 15 0 /tmp/cert.crt414800473 <nil>} {event 15 0 WRITE <nil>}]} {{info 2020-11-23 14:13:36.383533045 +0000 UTC m=+0.018927712  Loaded modified certificate undefined } [{certificate 15 0 /tmp/cert.crt414800473 <nil>} {event 15 0 WRITE <nil>}]} {{info 2020-11-23 14:13:36.38356223 +0000 UTC m=+0.018956858  Loading modified certificate undefined } [{certificate 15 0 /tmp/key.crt548122084 <nil>} {event 15 0 WRITE <nil>}]}]
--- FAIL: TestReload (0.03s)
```

--- a/pkg/config/tlscfg/cert_watcher_test.go
+++ b/pkg/config/tlscfg/cert_watcher_test.go
@@ -95,7 +95,8 @@ func TestReload(t *testing.T) {
 	waitUntil(func() bool {
 		// Logged when both matching public and private keys are modified in the cert.
 		// If mismatched keys are present in the cert, the "Failed to load certificate" error will be logged instead.
-		return logObserver.FilterMessage("Loaded modified certificate").Len() > 0
+		return logObserver.FilterMessage("Loaded modified certificate").
+			FilterField(zap.String("certificate", keyFile.Name())).Len() > 0
 	}, 100, time.Millisecond*200)
 
 	// Logged when the cert is modified with the client's public key due to

